### DISCUSSION TITLE
Add stacking capability to the Reporter bar graph

### DISCUSSION THREAD

**Discussion by kenderson4** [cortezaproject/corteza-server#1738] [Ideas]
Per the suggestion of @Fajfa in [the forum](https://forum.cortezaproject.org/t/stack-field-in-metrics-section-of-chart-builder/1892/8) I'm logging a feature request to add the stacking capability found in the Charts Section to bar charts in the the Reporter Section.

**Comment by Fajfa** [comment]
Created issue https://github.com/cortezaproject/corteza/issues/1739
